pkg/pillar/go.mod

```diff
@@ -60,3 +59,91 @@ replace github.com/lf-edge/eve/libs/zedUpload => ../../libs/zedUpload
 replace github.com/lf-edge/eve/libs/depgraph => ../../libs/depgraph
 
 replace github.com/lf-edge/eve/libs/reconciler => ../../libs/reconciler
+
+require (
```

Comment:
Btw. this second require block with indirect deps is a result of upgrading to Go 1. 17 (and not necessary a change in these deps): https://go.dev/doc/go1.17#go-command